src/desktop/apps/artwork/components/commercial/templates/index.jade

```diff
@@ -1,3 +1,6 @@
+- var newBuyNowFlow = user && user.hasLabFeature("New Buy Now Flow")
```

Comment:
We'll need to reuse it in both acquire and questions partials so moving it up here.

In general all of those partials are only seem to be used in this one spot below, so not sure how much we gain with all the partials break up here.